Server/src/controllers/dataUploadController.ts

```diff
@@ -0,0 +1,35 @@
+import { Request, Response } from 'express';
+import { DataProcessService } from '../services/dataProcess/dataProcessService'
+
+/**
+ * The dataUploadController is responsible for processing providing instructions to the application if a request comes in
+ * to the /dataupload api. This controller will call the Service for appropriate processing of the input. The
+ * controller is resposible for providing a response back to the Client on success. The fileUploadController 
+ * creates a request for the fileUploadService to process, only passing the path of the file.
+ */
+
+export class DataUploadController {
```

Comment:
I see al controllers have a createRequest method, consider putting it into an abstract class if such exist in typescript

Reply:
This is a good idea. Will flag this for future date